source/Calamari.AzureResourceGroup/DeployAzureResourceGroupBehaviour.cs

```diff
@@ -75,7 +75,7 @@ public async Task Execute(RunningDeployment deployment)
             Func<Task<IResourceManagementClient>> createArmClient = async () =>
                                                               {
                                                                   var token = new TokenCredentials(await ServicePrincipal.GetAuthorizationToken(tenantId, clientId, password, resourceManagementEndpoint, activeDirectoryEndPoint));
-                                                                  var resourcesClient = new ResourceManagementClient(token)
+                                                                  var resourcesClient = new ResourceManagementClient(token, AuthHttpClientFactory.ProxyClientHandler())
```

Comment:
What happens here if there is no proxy configured?
Does `WebRequest.DefaultWebProxy` being set to no proxy mean that it just works directly?

Reply:
Without a proxy configured, the ResourceManagementClient just makes calls regardless of the default proxy

Reply:
The images in the PR description show this